plugins/gossip/schedule.js

```diff
@@ -137,6 +137,13 @@ function (gossip, config, server) {
       })
   }
 
+  function currentlyDownloading () {
+    // don't schedule gossip if currently downloading messages
+    if (server.lastMessageAt && server.lastMessageAt > Date.now() - 500) {
+      console.log('skip gossip')
```

Comment:
Shouldn't this be: server.emit('log:info' ?

Maybe the message should be "already downloading messages from another peer, skip connect"